src/CMakeLists.txt

```diff
@@ -842,6 +854,18 @@ add_subdirectory(compressor)
 
 add_subdirectory(tools)
 
+# dmClock
+
+add_subdirectory(dmclock) # after gmock
+add_dependencies(tests dmclock-tests dmclock-data-struct-tests)
+
+if(WITH_TESTS)
+  install(PROGRAMS
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-tests
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-data-struct-tests
+    DESTINATION bin)
+endif(WITH_TESTS)
```

Comment:
Then somebody who wanted to build these binaries could pass `-DWITH_DMCLOCK_TESTS=ON` to cmake, but for purposes of building RPMs and debs we would use `-DWITH_DMCLOCK_TESTS=OFF`.